src/GafferScene/TweakPlug.cpp

```diff
@@ -210,136 +208,77 @@ IECore::MurmurHash TweakPlug::hash() const
 namespace
 {
 
-const char *modeToString( TweakPlug::Mode mode )
-{
-	switch( mode )
-	{
-		case TweakPlug::Replace :
-			return "Replace";
-		case TweakPlug::Add :
-			return "Add";
-		case TweakPlug::Subtract :
-			return "Subtract";
-		case TweakPlug::Multiply :
-			return "Multiply";
-		case TweakPlug::Remove :
-			return "Remove";
-		default :
-			return "Invalid";
-	}
-}
-
 // TODO - if these make sense, I guess they should be pushed back to cortex
 
 // IsColorTypedData
-template< typename T > struct IsColorTypedData : boost::mpl::and_< TypeTraits::IsTypedData<T>, TypeTraits::IsColor< typename TypeTraits::ValueType<T>::type > > {};
+template< typename T > struct IsColorTypedData : boost::mpl::and_< IECore::TypeTraits::IsTypedData<T>, IECore::TypeTraits::IsColor< typename IECore::TypeTraits::ValueType<T>::type > > {};
 
 // SupportsArithmeticData
-template< typename T > struct SupportsArithData : boost::mpl::or_<  TypeTraits::IsNumericSimpleTypedData<T>, TypeTraits::IsVecTypedData<T>, IsColorTypedData<T>> {};
+template< typename T > struct SupportsArithData : boost::mpl::or_<  IECore::TypeTraits::IsNumericSimpleTypedData<T>, IECore::TypeTraits::IsVecTypedData<T>, IsColorTypedData<T>> {};
```

Comment:
These changes are unnecessary now the implementation is back in the .cpp file.

Reply:
Fixed in 9b794ef88ab1ddd6435bc923e50cc3df76017da3